content/05.treat.md

```diff
@@ -180,28 +180,38 @@ However, in the long term, atomic convolutions may ultimately overtake grid-base
 
 #### *De novo* drug design
 
-*De novo* drug design attempts to model the typical design-synthesize-test cycle of drug discovery [@doi:10.1002/wcms.49; @doi:10.1021/acs.jmedchem.5b01849].
+*De novo* drug design attempts to model the typical design-synthesize-test cycle of drug discovery in-silico [@doi:10.1002/wcms.49; @doi:10.1021/acs.jmedchem.5b01849].
 It explores an estimated 10<sup>60</sup> synthesizable organic molecules with drug-like properties without explicit enumeration [@doi:10.1002/wcms.1104].
-To test or score structures, algorithms like those discussed earlier are used.
+To score molecules after generation or during optimization, physics-based simulation could be used [@tag:Sumita2018], but machine learning models based on techniques discussed earlier may be preferable [@tag:Gomezb2016_automatic], as they are much more computationally expedient. Computationally efficiency is particularly important during optimization as the "scoring function" may need to be called thousands of times.
+
 To "design" and "synthesize", traditional *de novo* design software relied on classical optimizers such as genetic algorithms.
-Unfortunately, this often leads to overfit, "weird" molecules, which are difficult to synthesize in the lab.
-Current programs have settled on rule-based virtual chemical reactions to generate molecular structures [@doi:10.1021/acs.jmedchem.5b01849].
+These approaches can lead to overfit, "weird" molecules, which are difficult to synthesize in the lab.
```

Comment:
Does this partly contradict the new text on genetic algorithms below?  What is our overall stance on genetic algorithms versus generative neural networks?

Reply:
That sentence I think was actually from the earlier version or it was accidently put here by mistake. It's funny because first autoencoder for molecular generation got the exact same criticism. In the papers on the ORGAN/ORGANIC GAN model they showed its' very easy to get "weird"/non-physical/impossible-to-synthesize molecules which are "over-optimized" for a certain thing if you don't put constraints in the loss function to prevent that.  Anyway, I would remove the sentence about 'weird' molecules.

Reply:
I agree, you can remove the sentence about "weird" molecules in your next round of edits.  Or alternatively note that this is still a problem with many NN-based methods.